phpunit.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit backupGlobals="false"
+         backupStaticAttributes="false"
+         colors="true"
+         convertErrorsToExceptions="true"
+         convertNoticesToExceptions="true"
+         convertWarningsToExceptions="true"
+         processIsolation="false"
+         stopOnFailure="false">
+    <testsuites>
+        <testsuite name="Application Test Suite">
```

Comment:
rename to Unit Test Suite, also maybe make this phpunit.xml.dist like the other repo.

Reply:
Name updated.
The other one is named .example b/c it's not complete (environment variable values needed) This one it makes sense to be checked in as is so anyone can start running unit tests.